### DISCUSSION TITLE
Twisted Wave Audio Editor

### DISCUSSION THREAD

**Discussion by Ballou** [elrumo/macOS_Big_Sur_icons_replacements#1469] [Icon request] [answered]
Would love if someone could take a stab at a replacement Twisted Wave icon. There is one currently available from user [peapodpost](https://macosicons.com/#/u/peapodpost), but it's a little bare, and strips much of the original's personality.

The app can be found at: https://twistedwave.com. Though he tirelessly maintains the app, esthetics don't appear to be a top priority of the developer, so he isn't offering anything (yet). 

Here is the current icon file: [TwistedWave.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/13112294/TwistedWave.icns.zip)

Thanks!

**Comment by vikasbhatia** [comment, marked as answer]
Hey, I gave this one a go:
<img width="256" alt="TwistedWave" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/84acb27a-3949-4f32-b511-5257965cf71d">
I submitted it for approval so I expect you'll be able to download it as an ICNS from my profile soon: https://macosicons.com/#/u/sidewinder

I re-created the original as well as I could from the OG icon, as I got the vibe that you wanted to stay true to it while converting it to the new shape. If you need any edits to it though, I'd be happy to make them for you.

**Comment by Ballou** [comment]
That’s pretty cool! I’ll certainly use it.

I’m not the developer, but I’m going to let him know about this in case he wants to be part of the conversation.

**Comment by vikasbhatia** [comment]
Sounds good! Glad I could help :)

PS. You can mark my comment as "Answered" to close the request.

**Reply by Ballou** [reply]
I emailed the developer. Shall I leave this open in case he wants to chime in?

**Reply by vikasbhatia** [reply]
For sure. It doesn’t make any difference to me, I only suggested it as it helps people filter open requests on the forum.

Since you’ve contacted the dev, let’s leave it until they reply.
